contracts/interfaces/IFeesEscrow.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+pragma solidity 0.7.5;
+
+/**
+ * @dev Interface of the FeesEscrow contract.
+ */
+interface IFeesEscrow {
+    /**
+    * @dev Event for tracking fees withdrawals to Pool contract.
+    * @param amount - the number of fees.
+    */
+    event FeesTransferred(uint256 amount);
+
+    /**
+    * @dev Function is used to transfer accumulated rewards to .
```

Comment:
Please update docstring. There is no admin role and "accumulated fees to Pool contract"